projects/plugins/protect/src/js/routes/firewall/index.jsx

```diff
@@ -576,54 +643,75 @@ const FirewallPage = () => {
 		</>
 	);
 
-	const bruteForceAllowListSettings = (
+	const ipAllowListSettings = (
 		<>
-			<div className={ styles[ 'brute-force-rules-section' ] }>
-				<Textarea
-					id="jetpack_waf_ip_allow_list"
-					label={ __( 'Always allowed IP addresses', 'jetpack-protect' ) }
-					description={
-						<>
-							<Text mb={ 1 }>
-								{ __(
-									"IP addresses added to this list will never be blocked by Jetpack's brute force protection.",
```

Comment:
Suggestion – can we keep the description for the allow list, i.e. "IP addresses added to this list will never be blocked by Jetpack's security features."